docs/serving/services/creating-services.md

```diff
@@ -1,76 +1,71 @@
 ---
 title: "Creating Knative services"
-weight: 10
+linkTitle: "Creating a service"
+weight: 02
 type: "docs"
 ---
 
-Knative Services are used to deploy an application. Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file. Every time the Configuration is updated, a new Revision is created.
+To create an application, you need to create a YAML file that defines a Knative service. This YAML file specifies metadata about the application, points to the hosted image of the app and allows the service to be configured.
```

Comment:
I'm not sure I like "to create an application", because there are lots of other ways to create an application. Maybe "To create an application with Knative, ..." ?

Reply:
I would lean more towards `deploy an application`